tools/ci/travis/travis-ci.sh

```diff
@@ -142,6 +142,8 @@ EOF
   if [ "$MIN_BUILD" != "1" ] ; then
     echo Running the testsuite with the debug runtime
     $MAKE USE_RUNTIME='d' OCAMLTESTDIR="$(pwd)/_ocamltestd" TESTLOG=_logd all
+    echo Ensuring that library documentation compiles
+    make -C ocamldoc manpages html_doc pdf_doc texi_doc
```

Comment:
The pdf_doc target requires pdflatex to be installed, isn't that a problem?

Reply:
It's only on Travis - I've pushed another commit to ensure the package is there (and only run this test on one of the jobs)